package.json

```diff
@@ -47,6 +47,7 @@
     "babel-loader": "^6.2.7",
     "babel-plugin-add-module-exports": "^0.2.1",
     "babel-plugin-external-helpers": "^6.18.0",
+    "babel-plugin-transform-object-rest-spread": "^6.16.0",
```

Comment:
Travis is complaining it can't find this babel plugin, but it's in the dependencies?